adapters/repos/db/filters_integration_test.go

```diff
@@ -977,7 +977,7 @@ var cars = []models.Object{
 			"horsepower":             int64(100),
 			"weight":                 1200.0,
 			"contact":                "sandra@efficientcars.example.com",
-			"description":            "This small car has a small engine and unicode labels (ąę), but it's very light, so it feels faster than it is.",
+			"description":            "This small car has a small engine and unicode labels (ąę), but it's very light, so it feels fater than it is.",
```

Comment:
You should be able to actually change this back to `faster`. I also investigated the reason for the failing test, and it's due [this filter](https://github.com/weaviate/weaviate/blob/master/adapters/repos/db/filters_integration_test.go#L523-L526) which expects the description of `carPolo` to match a specific length.

Adding the `s` to `fater` exceeds this expected length, so the id is not returned in the search. You can simply change this expected description length from `109` to `110`, and the test should pass. This way we can eliminate this typo as well 🙂

Reply:
Thanks.